doc/sphinx/language/core/sections.rst

```diff
@@ -114,3 +117,101 @@ usable outside the section as shown in this :ref:`example <section_local_declara
 
    Notice the difference between the value of :g:`x'` and :g:`x''` inside section
    :g:`s1` and outside.
+
```

Comment:
Let's not move this text.  A hyperlink somewhere in the first couple paragraphs of the chapter would be more appropriate, e.g. saying something like:

```The typing rules for sections are described :ref:`here <Admissible-rules-for-global-environments>.```

However, the current wording doesn't clearly suggest _any_ relationship to sections, so it probably needs a sentence or three explaining how it's relevant to sections.

The `CIC` chapter is dense and off-putting for those not already familiar with underlying math.  It needs significant work, in particular a subsection explaining the notation to those who don't know it and probably examples for almost all the rules.  Best to keep the mathematical underpinnings in one place.

Reply:
Mathematical descriptions for inductive types and modules are already moved.
I think that this PR is consistent with them.

However, I agree that adding a link is another choice to show the relationship between the section mechanism and its mathematical description.

Reply:
@Zimmi48 What do you think?

Reply:
> [coq/ceps#43](https://github.com/coq/ceps/pull/43) is not really relevant and the work envisioned then (~3 years ago) has been completed. While we move sections from time to time, each such move has to have a good rationale.

I disagree. The work envisioned in this CEP was never fully completed. For instance, I intended to explode the "Commands" chapter and move the various documented commands there in more relevant place. We also started splitting formal inference rules related to some concepts (modules) to the relevant chapter, so this proposed change seems consistent to me.

However, it would help to separate the move and the changes to the content in separate commits for better traceability.

Reply:
>I disagree. The work envisioned in this CEP was never fully completed.

Indeed, I meant only that the initial rearrangement of sections is done.  The CEP is more a statement of principles; I don't know that it will ever be clearly "done".  (How would others know that you  have intentions on exploding the "Commands" chapter? That doesn't seem to be in the text.)

But it is certainly irrelevant; anyone is free to suggest documentation changes at any time, which should be evaluated on the merits.